### PR TITLE
Add logging w decorators

### PR DESCRIPTION
**Pull request by DidierRLopes**
@piiq can you merge your PR into this one?

### CODE REVIEW

File: gamestonk_terminal/helper_funcs.py [hunk 1]
```diff
@@ -932,26 +929,44 @@ def get_rf() -> float:
...
-        if cfg.DEBUG_MODE:
-            return f(*args, **kwargs)
-        try:
-            return f(*args, **kwargs)
-        except Exception as e:
-            logger.exception("%s", type(e).__name__)
-            return []
-
-    return inner
+    def draw_lines_and_annotate(self):
```
Comment (DidierRLopes) [inline gamestonk_terminal/helper_funcs.py:936]: You got this repeated.

Also, can you remove that commented code on the function below :)

Comment (piiq) [reply gamestonk_terminal/helper_funcs.py:936]: aha

**Comment by DidierRLopes** [comment]
Thanks @piiq

**Comment by piiq** [comment]
will merge this to fix tests on main.

we will create a separate branch/PR for threading logging through the application

**Review by piiq — APPROVED** [review]

**Pull request merged**